### DISCUSSION TITLE
Odd looking window button in Wayland KDE

### DISCUSSION THREAD

**Discussion by Pestigirioso** [yokoffing/Betterfox#274] [Q&A] [answered]
Hi! I just installed this user.js file to improve my Firefox experience, and I noticed that the window buttons (i.e.: minimize, maximize, close) are odd, and invisible.

Is there any fix to this? 

Also, the bookmarks bar and it's icons look odd.

![image](https://github.com/yokoffing/Betterfox/assets/33265407/d9676520-38b8-4e76-a34c-1ab1981688f9)

![image](https://github.com/yokoffing/Betterfox/assets/33265407/b443a38d-e75f-42c1-8d1a-bafba758856d)

Thank you, and thanks for the great work!

**Comment by yokoffing** [comment]
I don't know of any pref that deals with the window buttons. Cosmetic alterations are usually done with a `userChrome` file.

@black7375, have you ever seen this before?

**Reply by black7375** [reply]
I haven't seen that yet.
In the case of bookmark icons, they may be cached and displayed after the site is accessed. (Privacy issues)

**Reply by Pestigirioso** [reply, marked as answer]
Hi! Indeed, it was a `userChrome` file. I think I had Firefox Nordic theme applied, and that messed up with the Betterfox user.js

I'll detail my solution, just in case:

1. Go to [about:profile](about:profile)
2. Open containing folder of the profile I was using
3. Locate the chrome folder and open it
4. Delete everything that was in there.

After that, both issues were fixed. Thank you guys for your time, and sorry for my dumbness :sweat_smile: 

You are really cool!
